src/shared/data/Tile-list.es6

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+//
+// Tile-lis
+// Created by dpekar on 10/7/15.
+//
+
+module.exports = {
+
+    'basic': [
```

Comment:
Hmm, maybe but I'm not quite sure what you have in mind.  I'm going to keep it this way for now but let's discuss live and happy to change :)